lib/cached_search_api_connection_response.rb

```diff
@@ -0,0 +1,3 @@
+# :nocov:
```

Comment:
@HalahAb , please add a spec for this instead of skipping code coverage.

Reply:
@MothOnMars, I added a test then had to remove it because that one line was still being seen as missing coverage. I couldn't figure out why and thought to remove it because it was only testing ruby struct functionality. I added the test back now, but it turns out it was the other struct definition in cached_search_api_connection.rb that was causing the coverage fail. I removed it and that seems to have fixed things.